riotfile.py

```diff
@@ -296,7 +307,8 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             venvs=[
                 Venv(pys="2.7"),
                 Venv(
-                    pys=select_pys(min_version="3.5"),
+                    pys=select_pys(min_version="3.5", max_version="3.10"),
+                    # depends on bytecode module, which doesn't yet support Python 3.11
```

Comment:
This means that we're not running any of the other internal tests against 3.11 due to bytecode.

There are a lot of tests in internal which are not dependent on `bytecode`. 


We could do this hack for now in `tests/internal/__init__.py`:

```python
import sys
import mock

if sys.version_info >= (3, 11, 0):
    sys.modules["bytecode"] = mock.MagicMock()
    import bytecode
```

not great but at least let's us run most of our tests still